scripts/check-openapi-changed.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
This is a script since it's easier to write multiple bash commands in a separate script than in the makefile.